tests/openwisp2/sample_users/migrations/0001_initial.py

```diff
@@ -20,8 +21,14 @@ class Migration(migrations.Migration):
 
     initial = True
 
+    dependency = (
+        '0012_alter_user_first_name_max_length'
+        if django_version()[:2] == '3.1'
+        else '0011_update_proxy_permissions'
+    )
+
     dependencies = [
-        ('auth', '0012_alter_user_first_name_max_length'),
+        ('auth', dependency),
```

Comment:
Why not just do `('auth', '0011_update_proxy_permissions')` if we don't depend on the next migration?